tests/clamav_client.rs

```diff
@@ -16,6 +16,25 @@ const SIZE_LIMIT_EXCEEDED_ERROR_RESPONSE: &[u8] = b"INSTREAM size limit exceeded
 
 fn assert_implements_send_sync<T: Send + Sync>(_t: T) {}
 
+/// When exceeding the configured size limit ClamAV might respond with:
+/// 1. `SIZE_LIMIT_EXCEEDED_ERROR_RESPONSE`
+/// 2. Stopping the connection via code 32 (BrokenPipe) or 104 (ConnectionReset)
```

Comment:
How come the connection never drops when I'm testing this locally? Is this behavior (ClamAV dropping the connection) documented somewhere?

Reply:
I've encountered this error on a Ubuntu virtual machine. Never happens locally though (macOs). I haven't checked if this is documented somewhere...